core/domain/skill_domain_test.py

```diff
@@ -69,6 +69,14 @@ def test_valid_misconception_id(self):
         self._assert_validation_error(
             'Expected misconception ID to be an integer')
 
+    def test_valid_misconception_name(self):
+        name = ('Lets generate a string of more than 60 characters'
+                'so that it will fail the test')
+        self.skill.update_misconception_name(0, name)
+        self._assert_validation_error(
+            'Expected misconception name to be less than or equal to 60')
+
+
```

Comment:
Only one newline require. 

(@oppia/dev-workflow-team I was expecting a lint test to catch this, is that correct?)